examples/cinematography_domain/schema.py

```diff
@@ -30,4 +37,4 @@ class Movie(DomainModel):
 
 # Render the schema to stdout when executed directly:
 if __name__ == "__main__":
-    print(cine_schema.as_str())
+    sys.stdout.write(cine_schema.as_str())
```

Comment:
Why wasn't `print` ok? Remember to flush it as well

Reply:
We have a linter which doesn't like `print` 🤷‍♂️ . Do we need to flush even if the process is exiting right away?

Reply:
Haha I see 😂 `noqa`?

Reply:
I've kept `print`s (and `pprint`s, which the linter also dislikes) in `examples` directory. But this here is actually a feature, to echo out the GQL schema (and not a debug / example / demo code). It looks more serious 🧐 with `stdout.write`  doesn't it? 😄

Reply:
If the print linter is causing too much pain (like this), please just remove it. The intention for adding it, is that I very often see people use print in debugging and forget to delete their print statements before commits.

Reply:
@doctrino I see the same thing (and also in my own code sometimes, admittedly 😁). My $0.02 would be to keep the linter in place. It really doesn't hurt. If we want it, just merge this in: https://github.com/cognitedata/cognite-gql-pygen/pull/11